tests/MatejTest.php

```diff
@@ -0,0 +1,51 @@
+<?php declare(strict_types=1);
+
+namespace Lmc\Matej;
+
+use Http\Mock\Client;
+use Lmc\Matej\Model\Command\ItemPropertySetup;
+use Lmc\Matej\Model\CommandResponse;
+
+/**
+ * @covers \Lmc\Matej\Matej
+ */
+class MatejTest extends TestCase
+{
+    /** @test */
+    public function shouldBeInstantiable(): void
+    {
+        $matej = new Matej('accountId', 'apiKey');
+        $this->assertInstanceOf(Matej::class, $matej);
+    }
+
+    /** @test */
+    public function shouldExecuteRequestViaBuilder(): void
```

Comment:
Ideally, I would like have integration test there, which will interact with the real API. However this is tricky right now, so lets keep this as an isolated integration test...